### ISSUE TITLE
brownie interprets input parameter `readCorrection` as input file

### ISSUE THREAD

**Issue by sean-la** [jfostier/brownie#16]
When I perform the following command:
```bash
./brownie readCorrection -t 18 -k ${kmer} -o ${outputDir}/illumina_corrected.fastq -p ${outputDir} ${short}
```
I get the following output:
```bash
Brownie: don't know how to open file: 'readCorrection'
Expected one of the following extensions: .fastq, .fasta, .sam, .raw (or .gz variants thereof)
```
Thanks in advance!

**Comment by gmiclotte** [comment]
Hello,

thanks for reporting this issue, there was a bug in the settings parsing, this should be resolved with commit f2f2ddb.

Kind regards,
Giles Miclotte

**Comment by sean-la** [comment]
That worked, thanks!
